vpr/src/power/power.cpp

```diff
@@ -1209,16 +1210,17 @@ void power_routing_init(const t_det_routing_arch* routing_arch) {
         int fanout_to_seg = 0;
         auto node = device_ctx.rr_nodes[rr_node_idx];
         t_rr_node_power* node_power = &rr_node_power[rr_node_idx];
+        const t_edge_size node_fan_in = rr_graph.node_fan_in(RRNodeId(rr_node_idx));
```

Comment:
we can use the ``t_edge_size``  when declaring the ``max_fanin``, ``max_IPIN_fanin``, ``max_seg_to_seg_fanout`` and ``max_seg_to_IPIN_fanout``, rather than the currently used ``int``. 
This can save the use of ``static_cast`` in downstream codes.

Reply:
+1